.gitignore

```diff
@@ -5,10 +5,20 @@ doc/tag/*
 # Ignore CMake build directory.
 build/
 
-#Ignore build artifacts
+# Ignore build artifacts
 *.o
 
 # Ignore code coverage artifacts
 *.gcda
 *.gcno
 *.gcov
+
+# Ignore CBMC proof artifacts
+**/cbmc/proofs/**/gotos
+**/cbmc/proofs/**/html
+**/cbmc/proofs/**/logs
+**/cbmc/proofs/**/TAGS-*
+**/cbmc/**/README.md
```

Comment:
I certainly would not ignore the README files.

Reply:
They were symbolically linked.